android/app/build.gradle

```diff
@@ -78,7 +78,7 @@ import com.android.build.OutputFile
  */
 
 project.ext.react = [
-    enableHermes: false,  // clean and rebuild if changing
+        enableHermes: true  // <- here | clean and rebuild if changing
```

Comment:
Remove `<- here |` part of the comment and fix indentation.

Reply:
done